lib/article_json/export/amp/elements/image.rb

```diff
@@ -6,6 +6,15 @@ class Image < Base
           include Shared::Caption
           include Shared::Float
 
+          # @param [ArticleJSON::Elements::Base] element
+          # @param String width
+          # @param String height
+          def initialize(element, width: '640px', height: '480px')
```

Comment:
I think it should not have the `px` in the value, all the examples don't have it (as a normal `img` tag wouldn't):
https://www.ampproject.org/docs/reference/components/amp-img

Reply:
And where do these values come from and how can they be overwritten? If they shouldn't be the AMP-standard values, I would maybe make them configurable through the gem config..

Reply:
@dsager right, changing it

Reply:
@nicolas-fricke  we still don't know, the thing is that for AMP is mandatory to specify the image width and height, and we don't store it in the json structure yet. 

So the plan was to hardcore the default values we use in devex for the images, have one implementation working and then think about how to make it configurable. But I thought having an initializer could be nicer and more useful than having it just hardcored.